docs/reference/api/docker_remote_api.md

```diff
@@ -24,6 +24,20 @@ client must have `root` access to interact with the daemon. If a group named
 `docker` exists on your system, `docker` applies ownership of the socket to the
 group.
 
+To interact with the remote API using [cURL](http://curl.haxx.se/download.html).
+You run `curl` against the socket where the Docker daemon is listening. For this
+reason, must use cURL 7.40 (or greater) as these versions have the
+`--unix-socket` flag available. To run `curl` against the deamon on
+the default socket, use the following:
+
+    curl --unix-socket /var/run/docker.sock http://containers/json
+
+If you bound the Docker daemon to a different, you would reference that
```

Comment:
bound to a different _what_? Port doesn't really make sense here. I don't think unix sockets have ports.